src/pie_modules/taskmodules/common/mixins.py

```diff
@@ -214,6 +214,24 @@ class RelationStatisticsMixin(StatisticsMixin[Dict[Tuple[str, str], int]]):
     skipped relations.
     """
 
+    def get_none_label_for_statistics(self) -> str:
+        if not hasattr(self, "_statistics_none_label"):
+            if hasattr(self, "none_label"):
+                # If the mixin has a `none_label` attribute, use it as the label for "no relation".
+                self._statistics_none_label = self.none_label
+            else:
+                self._statistics_none_label = "no_relation"
+                logger.warning(
+                    f"{type(self).__name__} does not have a `none_label` attribute. "
+                    "Using default value 'no_relation'. "
+                    "`none_label` is used as the label for relations with score 0 in statistics and "
+                    "all relations with label different from `none_label` will be summarized to 'all_relations'. "
+                    f"Set the `none_label` or `_statistics_none_label` attribute before using statistics or "
```

Comment:
don't recommend to overwrite a "private" attribute (`_statistics_none_label`). the other two recommendations (setting `non_label` or overwriting `get_none_label_for_statistics()`) should be sufficient.